images/scripts/arch.install

```diff
@@ -59,6 +64,7 @@ if [ -n "$do_install" ]; then
     packages=$(find build-results -name "*.pkg.tar.zst")
     pacman -U --noconfirm $packages
 
+    systemctl start firewalld
```

Comment:
Odd, how did that work before? The base image didn't change?

Reply:
This is indeed fishy,

images/scripts/arch.setup:systemctl enable --now firewalld